mycore-pi/src/main/java/org/mycore/pi/doi/client/datacite/MCRDataciteClient.java

```diff
@@ -136,34 +139,40 @@ public List<Map.Entry<String, URI>> getMediaList(final MCRDigitalObjectIdentifie
         HttpGet httpGet = new HttpGet(requestURI);
 
         try (CloseableHttpClient httpClient = getHttpClient()) {
-            CloseableHttpResponse response = httpClient.execute(httpGet);
-            StatusLine statusLine = response.getStatusLine();
-            switch (statusLine.getStatusCode()) {
-                case HttpStatus.SC_OK -> {
-                    try (Scanner scanner = new Scanner(response.getEntity().getContent(), StandardCharsets.UTF_8)) {
-                        while (scanner.hasNextLine()) {
-                            String line = scanner.nextLine();
-                            String[] parts = line.split("=", 2);
-                            String mediaType = parts[0];
-                            URI mediaURI = new URI(parts[1]);
-                            entries.add(new AbstractMap.SimpleEntry<>(mediaType, mediaURI));
+            MCRResultOrException<List<Map.Entry<String, URI>>, MCRPersistentIdentifierException> result
```

Comment:
is this correct? `List<Map.Entry<String, URI>`, could `Map<String,List<String>>`, ...,  but doesn't have to be fixed in this PR